src/components/SystemMessageNotification/SystemMessageNotification.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../theme";
+
+.warning{
+    border-radius: 0 !important;
+    background: #c60909;
+    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+    color: white;
+    padding-bottom: 1px;
+    margin-bottom: 0px;
+   /* position: absolute;
+    right: 0;
+    left:0;*/
+
+}
+
+.notice {
+    border-radius: 0 !important;
+    background: darkslategrey;
+    color: white;
+    padding-bottom: 2px;
+    margin-bottom: 0px;
+    /*position: absolute;
+    right: 0;
+    left:0;
+*/
+}
+
+.info {
+    border-radius: 0 !important;
+    background: #3871dc;
+    color: white;
+    padding-bottom: 2px;
+    margin-bottom: 0px;
+   /* position: absolute;
+    right: 0;
+    left:0;*/
+    }
```

Comment:
@AStiner  You are duplicating a lot of style between these message types. Create a `.alert` class to hold all the base styles and use the remaining types as only modifications (ex. background color).

Reply:
How did you apply the class? Did you apply both classes (the base class and the alert type)? Having a different name doesn't change specificity so you shouldn't be running into issues there. The order that you declare these classes matter.

Reply:
@jasekiw Base class? Do you mean the alert-container? It calls the alert-container then the Alert I believe. But, I tried to add that to the alert-container or alert and it did not make a change.